src/ngraph/runtime/cpu/cpu_external_function.cpp

```diff
@@ -826,8 +826,33 @@ using namespace ngraph::runtime;
                     }
                     return false;
                 };
-                // Always enable nodes computing output tensors
-                if (computes_output())
+
+                auto possibly_overwritten = [&]() {
```

Comment:
I'd suggest we move this to user `util.cpp` or `graph_util.cpp` It's a quite general routine, but I'm fine with keeping it as a lambda.